### PR TITLE
Adding XMLGregorianCalendar to LocalDate converters

### PR DESCRIPTION
**Pull request by antmordel**

### CODE REVIEW

**Comment by eric-martin** [comment]
@antmordel It doesn't look like our Jenkins build is handling java 9 modules yet. Can you undo your last change (the change I asked you to make) and I think we will be good to go.

Sorry about that.

Ref:
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.7.0:testCompile (default-testCompile) on project java-dates-2: Fatal error compiling: invalid target release: 1.9 -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :java-dates-2

**Comment by antmordel** [comment]
Hi @eric-martin. Sorry for not checking those errors. I rollbacked the changes. Also, I was thinking that we can still create the new module and just comment that one in the parent pom as we do with the `java-dates` one.

File: java-dates-2/pom.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                    <target>${maven.compiler.target}</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <properties>
+        <!-- testing -->
+        <assertj.version>3.6.1</assertj.version>
+		<maven.compiler.source>1.9</maven.compiler.source>
```
Comment (KevinGilmore) [inline java-dates-2/pom.xml]: indent with spaces instead of tabs

Comment (antmordel) [reply java-dates-2/pom.xml]: Thank you @KevinGilmore. It's been fixed.

**Pull request merged**